codelists/scripts/update_nhs_refsets.py

```diff
@@ -147,26 +189,47 @@ def get_latest_db_release_and_refset_tag_from_api(base_url):
         resp = requests.get(url)
         resp.raise_for_status()
         data = resp.json()
-        if SNOMED_ID in data and "database_alias" in data[SNOMED_ID]:
-            database_alias = data[SNOMED_ID]["database_alias"]
+        if coding_system_id in data and "database_alias" in data[coding_system_id]:
+            database_alias = data[coding_system_id]["database_alias"]
             # Get latest refset tag if available, or default to a known version if not
-            latest_refset_tag = data.get("pcd_refsets", {}).get("tag", "20250627")
+            latest_refset_tag = data.get(tag_id, {}).get("tag", "20250627")
             return database_alias, latest_refset_tag
         else:
-            print(f"No database_alias found for {SNOMED_ID} in API response.")
-            return None
+            print(f"No database_alias found for {coding_system_id} in API response.")
+            return None, None
     except Exception as e:
-        print(f"Error fetching latest DB release from API: {e}")
-        return None
-
-
-def build_temp_config(static_config_file, db_release, latest_tag):
-    with open(static_config_file) as f:
-        config = json.load(f)
+        print(e)
+        print(
+            f"Failed to connect to API at {url}, if this is a local instance ensure it is running."
+        )
```

Comment:
thought: As per the comment above about what the exception might be; it might not be a connection failure (even if that's likely), it could be malformed JSON (persistently bad data from the API).